src/partitionmanager.cpp

```diff
@@ -280,9 +280,11 @@ void PartitionManagerPrivate::refresh(const PartitionList &partitions, Partition
         if (partition->status == Partition::Mounted) {
             qint64 quotaAvailable = std::numeric_limits<qint64>::max();
             struct if_dqblk quota;
+            memset(&quota, 0, sizeof(quota));
+
             if (::quotactl(QCMD(Q_GETQUOTA, USRQUOTA), partition->devicePath.toUtf8().constData(), ::getuid(), (caddr_t)&quota) == 0
                     && quota.dqb_bsoftlimit != 0)
-                quotaAvailable = std::max((qint64)dbtob(quota.dqb_bsoftlimit) - (qint64)quota.dqb_curspace, 0LL);
+                quotaAvailable = std::max((qint64) dbtob(quota.dqb_bsoftlimit) - (qint64) quota.dqb_curspace, 0LL);
```

Comment:
nit: qt coding style guide says: "no space after cast" and "avoid C-style casts" -> deviates more after changes?

Reply:
Funny thing, the qtcreator clang-format rule I just applied to buteo-mtp specifically added such spaces. Could have been an interpretation that no space after c++ style cast, dunno. But maybe I'll just switch this to c++ casts.